sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py

```diff
@@ -103,6 +103,8 @@ def create_certificate(
             **kwargs
         )
 
+        kwargs.pop('tags', None)
```

Comment:
After some thought I prefer `tags = kwargs.pop("tags", None)` and passing it explicitly to `create_certificate`. As is this will make readers wonder why we're taking something out of kwargs here but not using it.

Reply:
makes sense, switching it to that way thanks!